src/esp/assets/BaseMesh.h

```diff
@@ -19,49 +25,141 @@
 namespace esp {
 namespace assets {
 
+/**
+  @brief Enumeration of mesh types supported by the simulator. Each entry
+  suggests a derived class of @ref BaseMesh implementing the specific storage
+  and processing interface to accomodate differing asset formats. Identifies the
+  derived variant of this object.
```

Comment:
I'm personally making the brief quite short, one sentence at most, putting the rest in a separate paragraph like so:

```cpp
/**
  @brief Enumeration of mesh types supported by the simulator. 

  Each entry suggests a derived class of @ref BaseMesh implementing the specific storage
  and processing interface to accomodate differing asset formats. Identifies the
  derived variant of this object.
*/
```

Reason is that the brief description is used in API listings and there you don't need the full thing.

Btw., I'm usually not indenting the main doc blocks in any way to have the full 80-char width available for text. (Not sure if clang format agrees with me on that, tho.)

Reply:
Great point, thanks! 😄